dae/dae/dask/client_factory.py

```diff
@@ -32,8 +33,8 @@ def add_arguments(parser):
             "kubernetes workers")
         group.add_argument(
             "--container-image",
-            default="registry.seqpipe.org/seqpipe-gpf:"
-            "dask-for-hist-compute_fc69179-14",
+            default="registry.seqpipe.org/iossifovlab-gpf:"
+            "import-k8s-support_bee6c29-7",
```

Comment:
This default value is not good. It should be changed before merging.

Reply:
Fixed!